src/test/tests.utils.ts

```diff
@@ -116,7 +132,12 @@ export function randomSnapshot(vaultDefinition?: VaultDefinition): IVaultSnapsho
 
 export function randomVault(chain?: Chain): VaultDefinition {
   const definitions = (chain ? [chain] : loadChains()).flatMap((chain) => chain.vaults);
-  return definitions[Math.floor(Math.random() * definitions.length)];
+
+  const controlledDefs = definitions.filter((vault) => {
```

Comment:
it's impossible to have all tokens mocked, so i left only those valts in `randomVault`, with tokens that we have in mocked map